### ISSUE TITLE
allow mergeSequenceTables to accept a vector of files

### ISSUE THREAD

**Issue by rec3141** [benjjneb/dada2#438]
instead of doing
```
st1 <- readRDS("/path/to/run1/output/seqtab.rds")
st2 <- readRDS("/path/to/run2/output/seqtab.rds")
st3 <- readRDS("/path/to/run3/output/seqtab.rds")
st.all <- mergeSequenceTables(st1, st2, st3)
```

for the purposes of reproducibility it would be convenient to be able to do this and avoid manual editing of the script each time:

```
st.all <- mergeSequenceTables(list.files(fq.path,pattern="*.rds$"))
```

**Comment by benjjneb** [comment]
This can be achieved through a bit of R hacking to get the function call into the right form. I think the following should work:

```
do.call(mergeSequenceTable, list(list.files(fq.path,pattern="*.rds$")))
```

**Comment by rec3141** [comment]
yes it should work but the problem is that the second table is required

```
do.call(mergeSequenceTables, list(list.files(fq.path,pattern="*.rds$")))
Error in (function (table1, table2, ..., repeats = "error", orderBy = "abundance")  :
  argument "table2" is missing, with no default
```

**Comment by fanli-gcb** [comment]
Why use `...` when a single vector of filenames could be provided? IMO that's the usual case instead of having to hack the function call. Worth keeping `table1` and `table2` arguments to retain backwards compatibility?

**Comment by rec3141** [comment]
ps here is my current hack, which seems to work fine

```
plate.list <- list.files(fq.path,pattern="*.rds$")
st.all <- readRDS(plate.list[1])
for(i in 2:length(plate.list)) {
st1 <- readRDS(plate.list[i])
st.all <- mergeSequenceTables(st1,st.all)
}
```

**Comment by benjjneb** [comment]
Fixed in #440 

Thanks to @fanli-gcb
